draft-packages/form/KaizenDraft/Form/Primitives/InputRange/styles.scss

```diff
@@ -0,0 +1,159 @@
+@import "~@kaizen/design-tokens/sass/color";
+
+// The draghandle
+$thumb-color: var($color-blue-500-id, $color-purple-600);
+$thumb-radius: 100%;
+$thumb-border-height: 2px;
+$thumb-height: 26px;
+$thumb-height-with-border: $thumb-height + ($thumb-border-height * 2);
+$thumb-width: 26px;
+$thumb-hover-shadow-color: var($color-blue-600-id, $color-purple-700);
```

Comment:
Are these `var($blah-id, $blah)` old "decision tokens" and should they be so?

Reply:
Oops yep, good catch